demos/meltdown_of.cc

```diff
@@ -28,17 +28,25 @@
 #  error Unsupported architecture. IA32 required.
 #endif
 
+#include <signal.h>
+
 #include <array>
 #include <climits>
 #include <cstring>
 #include <iostream>
 
-#include <signal.h>
-
 #include "cache_sidechannel.h"
 #include "instr.h"
-#include "meltdown_local_content.h"
+#include "faults.h"
 #include "local_content.h"
+#include "utils.h"
+
+// Different platforms raise different signals in response to the OF trap.
+#if SAFESIDE_LINUX
+const int kOverflowSignal = SIGSEGV;
```

Comment:
nit: constexpr: https://google.github.io/styleguide/cppguide.html#Static_and_Global_Variables "Constant initialization of static storage duration variables should be marked with constexpr or where possible the `ABSL_CONST_INIT` attribute. Any non-local static storage duration variable that is not so marked should be presumed to have dynamic initialization, and reviewed very carefully."

(There's no downside to using constexpr here, since all const ints that are initialized with a constant-expression are actually equivalent to constexpr ints, so the cons in https://google.github.io/styleguide/cppguide.html#Use_of_constexpr don't apply.)


BTW total opinion, but you could also replace the use of the C preprocessor with a compile-time `?:`:

```c++
constexpr int kOverflowSignal = SAFESIDE_MAC ? SIGFPE : SIGSEGV;
```

(but I think most people would be fine with it as written.)

Reply:
Fixed.